src/profile/style.css

```diff
@@ -157,6 +157,19 @@ nav h1::before {
   border-right: 1px solid #c4c4c4;
 }
 
+.oldDiff {
+  background-color: #fd5757;
```

Comment:
Please Use variables instead of directly assigning to hex

Reply:
Can you please check now?